tests/utils.py

```diff
@@ -793,13 +793,6 @@ def test_case(self):
             spans[i].assert_matches(parent_id=self.span_id, trace_id=self.trace_id, _parent=self)
             spans[i].assert_structure(root, _children)
 
-    def pprint(self):
```

Comment:
Was this not used?

Reply:
I could not find a place where `TestSpanNode.pprint()` was used so I just removed this test. We used `TestSpan.pprint()` in a few places but that method was inherited from Span.

Reply:
There were 4 places where we used `span.pprint` to log a span after an assertion failed. Instead of renaming the deprecated method I just removed it altogether `Ex: assert span.metrics.get("out.port") -- , span.pprint() -- `